resources/css/style.css

```diff
@@ -98,3 +98,162 @@ a:visited {
   gap: 2rem;
   margin-top: 1.2rem;
 }
+
+/* WORKS SECTION */
+
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(4, 1fr);
+  margin: 11.4rem 2.4rem 0;
+}
+
+.works__tonic-mps {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #fff;
+  padding: 1.6rem;
+  margin-bottom: 8.8rem;
+  border: 1px solid #dfe1e6;
+  border-radius: 1.6rem;
+}
+
+.works__image {
+  max-width: 100%;
+}
+
+.works__heading {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 3.2rem;
+  line-height: 4.4rem;
+}
+
+.works__list--1 {
+  color: #7a869a;
+  line-height: 1.6rem;
+  display: flex;
+  justify-content: baseline;
+  gap: 1.2rem;
+  margin: 2rem 0;
+}
+
+.canopy {
+  color: #344563;
+  font-weight: 600;
+  font-size: 1.3rem;
+  text-transform: uppercase;
+}
+
+.dot {
+  height: 8px;
+  width: 8px;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+  display: inline-block;
+}
+
+.works__paragraph {
+  color: #344563;
+  font-weight: 400;
+  font-size: 1.5rem;
+}
+
+.works__list--2 {
+  display: flex;
+  gap: 3.2rem;
+  color: #6070ff;
+  margin-top: 1.2rem;
+}
+
+.works__list--2-item {
+  background-color: #ebebff;
+  padding: 0.4rem 1.2rem;
+  border-radius: 0.8rem;
+}
+
+.btn {
+  margin-top: 4rem;
+  margin-bottom: 4rem;
+}
+
+.btn-primary {
+  padding: 1.2rem;
+  color: #396df2;
+  border: 1px solid #6070ff;
+  border-radius: 0.8rem;
+  font-size: 1.7rem;
+  font-weight: 500;
+  line-height: 2.4rem;
+}
+
+.btn-primary:hover {
+  color: #fff;
+  background-color: #6070ff;
+}
+
+.btn-primary:active {
+  color: #fff;
+  background-color: #2230d2;
+}
+
+.btn-primary:visited {
+  color: #5e6c84;
+  border: 1px solid #c1c7d0;
+}
```

Comment:
[OPTIONAL] The previous reviewer accidentally missed this requirement, therefore this requirement is optional, however, had this been a first review this comment would've been a [REQUIRED] change. I strongly encourage you to implement the changes I'm suggesting.

Great job matching the template, very impressive project overall! Since you seem to be a little more experienced, I'd like to push you a little harder.

Take a look at your button designs:

Regular
<img width="132" alt="Screen Shot 2022-05-25 at 13 30 32" src="https://user-images.githubusercontent.com/82426682/170340429-bccbe252-fa39-40c8-be74-7a24afc1def1.png">

Hover
<img width="129" alt="Screen Shot 2022-05-25 at 13 31 05" src="https://user-images.githubusercontent.com/82426682/170340625-d451a687-3c0c-4405-a99d-9c3559744491.png">

Active
<img width="132" alt="Screen Shot 2022-05-25 at 13 35 20" src="https://user-images.githubusercontent.com/82426682/170341945-e6861c3c-3913-45c2-bafd-6379ded7cb37.png">

And now take a look at the template design:
<img width="629" alt="Screen Shot 2022-05-25 at 13 36 18" src="https://user-images.githubusercontent.com/82426682/170342249-e323cb0d-d2cb-4eb6-a38f-d2ed1077a574.png">

Not quite the same, are they? I challenge you to tweak your button design so it looks just like the template design 💪

Reply:
@vechicin  I see the problem. I forgot to change the background color to white when visited. If you see in my code the hover interactions work until you click on them it goes to the disable mode, which is the grey bottom with a white background. Should I remove the disabled mode? or is there a different pseudo-class I should use? Thanks a lot for your help!

Reply:
I recommend you still implement the disabled styling.